schemas/app-sre/app-1.yml

```diff
@@ -26,6 +26,7 @@ properties:
     - TransitionPeriod # The app is proving SLI/SLOs for a determined period of time
     - OnBoarded        # The app is fully on-boarded, App SRE are carrying the pager
     - BestEffort       # The app is supported on a best effort basis
+    - PreApproval      # The app is not yet approved for onboarding
```

Comment:
PreApproved or PreApproval?  Let's make the PR comment and what is added match.

Reply:
i though PreApproval makes sense because its a noun? you tell me, you are the english speaker :)

Reply:
+1 to reusing Proposed.